test/rulesets/LinearAlgebra/dense.jl

```diff
@@ -1,8 +1,3 @@
-function generate_well_conditioned_matrix(rng, N)
-    A = randn(rng, N, N)
-    return A * A' + I
-end
-
```

Comment:
why don't we need this anymore?

Reply:
we do. it just moved.

before it was in a LinearAlgebra test file, which meant the files had to run in a certain order -- i moved it to the `test_utils` file so that the order of tests no longer matter

Reply:
Oh found it, it moved